src/ontology/hp.Makefile

```diff
@@ -369,19 +369,13 @@ hpoa:
 	
 	echo "##### HPOA: Running Make pipeline"
 	cd $(RARE_DISEASE_DIR)/ \
-		&& echo "##### HPOA: Running MISC Makefile" \
-		&& $(MAKE) -C misc \
 		&& echo "##### HPOA: Running CURRENT Makefile" \
 		&& $(MAKE) -C current
-	cd $(RARE_DISEASE_DIR)/util/ \
-		&& echo "##### HPOA: Running ANNOTATION Makefile (UTIL)" \
-		&& $(MAKE) -C annotation
 	
 	echo "##### HPOA: COPYING all result files into HPOA results directory"
 	mkdir -p $(HPOA_DIR)
-	cp $(RARE_DISEASE_DIR)/util/annotation/genes_to_phenotype.txt $(HPOA_DIR)
-	cp $(RARE_DISEASE_DIR)/util/annotation/phenotype_to_genes.txt $(HPOA_DIR)
-	cp $(RARE_DISEASE_DIR)/misc/*.tab $(HPOA_DIR)
```

Comment:
We have been releasing

[phenotype_annotation.tab](https://github.com/obophenotype/human-phenotype-ontology/releases/download/v2023-01-27/phenotype_annotation.tab)

and

[phenotype_annotation_negated.tab](https://github.com/obophenotype/human-phenotype-ontology/releases/download/v2023-01-27/phenotype_annotation_negated.tab)

What about these files? Are they deprecated from now on?

Reply:
Yes these are no longer in the pipeline and have't been used for awhile. Some external folks may still use them but these were deprecated in 2021 in favor of phenotype.hpoa.

Also, with the recent merge of monarch-initiative/hpo-annotation-data#876 misc and util directories no longer exist.

Reply:
Yeah I know, I am just saying that we remove a release file, but it makes sense if it was already depreacted.